index.js

```diff
@@ -7,12 +7,74 @@ function ask(questionText) {
   });
 }
 
-start();
+//random number generator function
+function randomNumber(min, max) {
+  let range = max - min + 1;
+  return Math.floor(Math.random() * range) + min;
+}
+
+//function to update the guess
+function updateGuess(min, max) {
+  return Math.floor((max - min) / 2 + min);
+}
+
+let guess = randomNumber(1, 100);
+let min = 1;
+let max = 100;
+
+start(); //calls function to start the game
 
+//async function to start the game
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  //computer asks for a max range higher than 1
+  max = await ask(
+    "Please enter a number for the maximum that is greater than 1. "
+  );
+  //console logs greeting to start the game
+  console.log(
+    `Let's play a game where you (human) pick a number between ${min} and ${max} and I (computer) try to guess it.`
+  );
+  //awaits user (human) to enter a secret number
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n"
+  );
+  //console logs the secret number that user inputs
+  console.log("You entered: " + secretNumber);
+  secretNumber = parseInt(secretNumber);
+  //console.log(typeof secretNumber);
+  //computer guesses number between min-max range
```

Comment:
Currently, `guess` is still a value between 1 and 100. When I set my max to 10, it guessed 17. You can take this a couple different ways, either utilizing your `updateGuess` sooner OR passing `min` and `max` into `randomNumber` at the initialization of `guess.`